assets/tests/consolidated_1_1_tests.yaml

```
@@ -3537,7 +4129,49 @@ tests:
             expectation:
               - module:a
               - module:b
-
+  #        listUsersAssertions:
```

It's important to annotate the tests that are skipped with _why_ they're skipped. This way, when that functionality is added/patched, we can uncomment in those respective PRs. 

```suggestion
  #  listUsersAssertions are commented-out because ....
  #        listUsersAssertions:
```